src/Auth.php

```diff
@@ -284,7 +284,7 @@ public function connection_imap($host, $login, $pass)
      *
      * @return false|array
      */
-    public function connection_ldap($ldap_method, $login, $password, ?bool &$error = null)
+    public function connection_ldap($ldap_method, $login, $password, ?bool &$error = null) // @phpstan-ignore parameterByRef.unusedType (phpstan doens't see that this value can be null - unsure why)
```

Comment:
I am fairly sure using null will always throw a PHP warning. You cannot have a reference to null. Do we know why this parameter is nullable instead of simply having a true/false state?